tests/test_schema.py

```diff
@@ -0,0 +1,37 @@
+"""Tests for the table JSON schema file."""
+
+import json
+from pathlib import Path
+
+import pytest
+from jsonschema import Draft4Validator
+
+_TESTS_PATH = Path(__file__).parent
+_REPO_PATH = _TESTS_PATH.parent
+_SCHEMA_PATH = _REPO_PATH / "keyFiles" / "table_schema.json"
```

Comment:
So `table_schema.json` is not bundled with the repository? Is the table_schema not actually used at any point?

Also, it's probably worth including `tests/data` as a fixture in `conftest.py` to not need to do this both here and in the regression tests

Reply:
Nope, it's not actually used by the code. I guess that makes sense -- you only really need to validate input data and `table_schema.json` describes an output file. I assume the reason they made it was essentially because they were trying to define a kind of standard for this data. It probably is quite handy as a reference if you're writing a script to process these files.

Good point about the fixture. I'll do that.